main.js

```diff
@@ -1127,9 +1136,9 @@ $("#soft-reset").click(function (event) {
 });
 
 function guessModelFromUrl() {
-    if (window.location.hostname.indexOf("bbc") === 0) return "B";
+    if (window.location.hostname.indexOf("bbc") === 0) return "B-DFS1.2";
```

Comment:
Love it, thanks! the terrible hack that means `master.godbolt.org` should select the right model :)